src/client/kotlin/tech/thatgravyboat/skycubed/features/overlays/DialogueOverlay.kt

```diff
@@ -113,6 +147,8 @@ object DialogueOverlay : Overlay {
     }
 
     private fun reset() {
+        lastClickedEntities =
+            lastClickedEntities.filterValues { it > System.currentTimeMillis() - 20000 }.toMutableMap()
```

Comment:
instead of 20k this should be the max display time of either action or non action messages using the config value plus maybe like an extra 10s for latency